src/main/webapp/stylesheets/teammatesCommon.css

```diff
@@ -1173,7 +1173,8 @@ Glyphicon for success status messages
 .alert-success:before {
     font-family: 'Glyphicons Halflings';
     display: inline-block;
-    padding-right: 0.5em;
+    padding-top: .2em;
```

Comment:
Haha @LiHaoTan is always for solving the root problem. I believe he wants to ensure `alert` classes are never used in modals in the first place, but create custom classes for modals if we want to colour them.